app/main.go

```diff
@@ -104,6 +104,7 @@ type options struct {
 		Spam    string `long:"spam" env:"SPAM" default:"this is spam" description:"spam message"`
 		Dry     string `long:"dry" env:"DRY" default:"this is spam (dry mode)" description:"spam dry message"`
 		Warn    string `long:"warn" env:"WARN" default:"You've violated our rules and this is your first and last warning. Further violations will lead to permanent access denial. Stay compliant or face the consequences!" description:"warning message"`
+		Restore string `long:"restore" env:"RESTORE" default:"" description:"restore message"`
```

Comment:
Same as in the Readme, should be updated to describe better what setting this variable does.